src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(100);
```

Comment:
```suggestion
        ball.setNumber(MAX_NUMBER);
```
avoid magic numbers - create a constant (private static final)